### PR TITLE
`kubernetes_cluster/node_pool`- `capacity_reservation_group_id`

### PR DESCRIPTION
**Pull request by heiazuo**
### public pr for `kubernetes_cluster/node_pool`- `capacity_reservation_group_id`

### CODE REVIEW

File: website/docs/r/kubernetes_cluster.html.markdown [hunk 1]
```diff
@@ -312,6 +312,8 @@ A `default_node_pool` block supports the following:
 
 * `vm_size` - (Required) The size of the Virtual Machine, such as `Standard_DS2_v2`. Changing this forces a new resource to be created.
 
+* `capacity_reservation_group_id` - (Optional) AKS will associate the specified agent pool with the Capacity Reservation Group. Changing this forces a new resource to be created.
```
Comment (tombuildsstuff) [inline website/docs/r/kubernetes_cluster.html.markdown:315]: presumably this should be:

```suggestion
* `capacity_reservation_group_id` - (Optional) Specifies the ID of the Capacity Reservation Group within which this AKS Cluster should be created. Changing this forces a new resource to be created.
```

File: website/docs/r/kubernetes_cluster_node_pool.html.markdown [hunk 2]
```diff
@@ -71,6 +71,8 @@ The following arguments are supported:
 
 ---
 
+* `capacity_reservation_group_id` - (Optional) AKS will associate the specified agent pool with the Capacity Reservation Group. Changing this forces a new resource to be created.
```
Comment (tombuildsstuff) [inline website/docs/r/kubernetes_cluster_node_pool.html.markdown:74]:
```suggestion
* `capacity_reservation_group_id` - (Optional) Specifies the ID of the Capacity Reservation Group where this Node Pool should exist. Changing this forces a new resource to be created.
```

**Review by tombuildsstuff — CHANGES_REQUESTED** [review]
hi @heiazuo 

Thanks for this PR, I've taken a look through and left a couple of comments inline but if we can fix those up then we can take another look.

Thanks!

**Comment by heiazuo** [comment]
Thanks @tombuildsstuff , documents updated, please take another look.

**Review by katbyte — APPROVED** [review]
LGTM 🦕

**Pull request merged**